link_shortener/form_routes.py

```diff
@@ -29,15 +29,16 @@
 
 
 class CreateForm(SanicForm):
-    endpoint = StringField('Endpoint', validators=[DataRequired()])
-    url = StringField('URL', validators=[DataRequired()])
+    endpoint = StringField('Endpoint', validators=[DataRequired(), NoneOf('/')])
+    url = StringField('URL', validators=[DataRequired(), NoneOf('/')])
     password = PasswordField('Password')
     switch_date = DateField('Status switch date')
     submit = SubmitField('Create')
 
 
 class UpdateForm(SanicForm):
-    url = StringField('URL', validators=[])
+    endpoint = StringField('Endpoint', validators=[DataRequired(), NoneOf('/')])
```

Comment:
No všeobecně počítáme s tím, že se nás nikdo nebude pokoušet "hackovat" když je to všechno za OAuthem, proto třeba nečistíme data. Ale tohle je asi v pohodě, minimálně u endpointu. U URL mi to moc smysl nedává, ale můžeš mi zkusit popsat situaci kdy by se to hodilo

Reply:
Neslo mi ani o hackovanie, ale o pripad, ked user sa rozhodne nechat adresu ako nas landing page.
Mas pravdu, asi nemusime obmedzovat URLko - nech si user tam dava co chce.

Reply:
fixnute
0e690f79e46358eafc87b9a27c23bd6b4f1642cf